include/dev/rtl8139_reg.h

```diff
@@ -0,0 +1,22 @@
+#ifndef _RTL8139_REG_H_
+#define _RTL8139_REG_H_
+/*
+ * Copied from FreeBSD: sys/dev/rl/if_rlreg.h
+ */
+
+#define RL_COMMAND 0x0037 /* command register */
```

Comment:
The layout of this header file is strange. You labeled the definition of `RL_CMD_RESET` with notable comment
```C
/*
 * Command register.
 */
```
but totally separated it from the definition of `RL_COMMAND`. Why not
```C
/*
 * Command register.
 */
#define RL_COMMAND 0x0037
#define RL_CMD_RESET 0x0010
```

Reply:
as stated in comment - it is copied from freebsd - I only removed not needed constants.
I think in such cases it is expected to do not reorder registers - that approach is useful if we want to use difftool in future.